### PR TITLE
some docs cleanup

### PR DESCRIPTION
**Pull request by shiftkey**
The team had a catch up yesterday about a bunch of things, and one of the topics was how to handle stale Pull Requests to keep things manageable. I took this as an opportunity to revisit the docs around PR reviews and refresh the page to make it more like our Issue Triage docs.

 - [Rendered **Pull Request Triage** doc](https://github.com/desktop/desktop/blob/docs-cleanup/docs/process/pull-request-triage.md)
 - [Rendered **Notes for Contributors** doc](https://github.com/desktop/desktop/blob/docs-cleanup/docs/process/notes-for-contributors.md)

I think that **Notes for Contributors** can eventually live somewhere else, but for now I'm going to leave it there as it's not really related to this PR.

### CODE REVIEW

File: docs/process/pull-request-triage.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+  change introduced unexpectedly on `master` but would affect everyone when they
+  updated their local repositories.
+
+These should be called out by the merging person with a `#jolo` - "Josh Only
+Lives Once" - to acknowledge to everyone that they're bypassing the review
+process, and the reasons for doing that.
+
+### Stale Pull Requests
+
+A reviewer will return to an **open and reviewed** pull request if, after 14
```
Comment (j-f1) [inline docs/process/pull-request-triage.md]: Should this mention that the PR will be closed?

Comment (shiftkey) [reply docs/process/pull-request-triage.md]: I'll add a note to the bottom of the doc about this - we don't want closing the PR to be interpreted the wrong way.

File: docs/process/pull-request-triage.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+
+### Everyone Reviews
+
+While everyone has their own domain expertise around the codebase, we encourage
+people to share the load of reviews and reviewing areas of the codebase that
+aren't as familiar. This spreads knowledge across the team
+
+### 24 Hours Cooling Off
+
+After being approved, most contributions will remain in this state for 24 hours
```
Comment (nerdneha) [inline docs/process/pull-request-triage.md]: for **at least** 24 hours?

File: docs/process/pull-request-triage.md [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+- [#2733](https://github.com/desktop/desktop/pull/2733) was pinning a dependency
+  that affected our continuous integration tests by installing the incorrect
+  version
+
+- [#4319](https://github.com/desktop/desktop/pull/4319) was a minor packaging
+  change introduced unexpectedly on `master` but would affect everyone when they
+  updated their local repositories.
+
+These should be called out by the merging person with a `#jolo` - "Josh Only
+Lives Once" - to acknowledge to everyone that they're bypassing the review
```
Comment (nerdneha) [inline docs/process/pull-request-triage.md]: lol

File: docs/process/pull-request-triage.md [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+We encourage a strong review culture, and contributors should not merge their
+own PRs unless there are exceptional reasons.
+
+Examples of exceptional situations:
+
+- [#2733](https://github.com/desktop/desktop/pull/2733) was pinning a dependency
+  that affected our continuous integration tests by installing the incorrect
+  version
+
+- [#4319](https://github.com/desktop/desktop/pull/4319) was a minor packaging
```
Comment (nerdneha) [inline docs/process/pull-request-triage.md]: loving that you're including these examples!

**Comment by shiftkey** [comment]
> Love it! Should we also add a paragraph about it being included in a release? Something about how it gets queued up and we attempt to release monthly (but no guarantees).

:thumbsup: might make a mention of that here and then follow up with refreshing our release notes to talk about that in more detail.

**Comment by shiftkey** [comment]
👽

**Comment by iAmWillShepherd** [comment]
Gonna go ahead and merge this to reduce our outstanding PR count.

**Pull request merged**